Assets/Scripts/Utility/Table.cs

```diff
@@ -376,6 +386,20 @@ public int GetInt(string columnName, string key)
 
         #endregion
 
+        #region Public Helpers
+
+        /// <summary>
+        /// Ensures that a string can be used as a value for a table entry.
+        /// </summary>
+        /// <param name="text">A candidate for an entry value.</param>
+        /// <returns>A safe string.</returns>
+        public static string SanitizeEntryValue(string text)
```

Comment:
Something I've been meaning to do for a while is add extra intelligence to Table for supporting string literals so that commas don't break the schema. Would it help you if I bring this forward?

Reply:
It would be great! I'm curious to know what you have in mind. I imagine they could be enscaped with a slash (no changes needed to current tables) or the whole string could be put inside quotes (a bit more readable).

Reply:
Cheers, I'll work on that sooner then. :) My thinking was just to enclose literals in quotes and expect literals to be declared in the schema. For example:

    schema: *id, group ,$notes
    0, widgets, "These are quality widgets, yes really!"

The reason for declaring string literals in schema is that table might otherwise want to use double-quotes for some other reason. I don't want to assume that or we end up with same issue comma has now.

Will that get the job done?

Reply:
I updated my comment at the same time you posted yours but i got my answer anyway ahah
I agree leaving the choiche to the schema is the safest option 👍